yt_dlp/extractor/archiveorg.py

```diff
@@ -241,3 +251,87 @@ def _real_extract(self, url):
                     'parent': 'root'})
 
         return info
+
+
+class YoutubeWebArchiveIE(InfoExtractor):
+    IE_NAME = 'youtube:web.archive.org'
+    IE_DESC = 'web.archive.org saved youtube videos'
+    _VALID_URL = r"""(?x)^
+                (?:https?://|//)?web\.archive\.org\/
+                    (?:web/)?
+                    (?:[0-9A-Za-z_*]+/)? # /web and the version index is optional
```

Comment:
```suggestion
                    (?:[0-9A-Za-z_*]+/)?  # /web and the version index is optional
```
Flake8 doesnt detect this since it is inside the regex, but use 2 spaces before comments. Similar below